src/temporal-logic/normalize_property.cpp

```diff
@@ -103,6 +103,14 @@ exprt normalize_property_rec(exprt expr)
     op = normalize_property_rec(op); // recursive call
 
   // post-traversal
+  if(expr.id() == ID_R)
+  {
+    if(to_R_expr(expr).lhs().is_false())
+    {
+      // false R ψ ≡ G ψ
```

Comment:
normalize_property.h documents the collection of normalisation steps applied, so I think this should be added to that documentation.

Reply:
added